### ISSUE TITLE
You need update GPU usage, please

### ISSUE THREAD

**Issue by NeiroNext** [BrainJS/brain.js#869]
## _What_ is wrong?

Brain JS use old version of GPU.js with  gpu_js class, now gpu library has only GPU class, not gpu_js.
Need only overwrite all gpu_js.* to GPU.* and gpu_js.GPU to GPU and  all will be work, now GPU mode not work.
Thanks!

**Comment by Aarstad** [comment]
this is correct, i managed to work around this by adding this  in my own script tag (before my js code)
```html
<script crossorigin="anonymous" src="https://unpkg.com/gpu.js@2.9.5/dist/gpu-browser.min.js"></script>
<script>
    gpu_js = GPU;
    gpu_js.GPU = GPU;
</script>
```

I know i know, i should use package manager etc.. just having some fun!

**Comment by Vectorrent** [comment]
Is this still an issue for anyone? As of beta v23, I am able to successfully train and run inference with the NeuralNetworkGPU() class, in both Firefox and Chromium.
